x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -32,7 +32,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const ruleName2 = data[1].rule.name;
 
   // Failing: See https://github.com/elastic/kibana/issues/147998
-  describe.skip('Findings Page', () => {
+  describe('Findings Page', () => {
```

Comment:
please remove changes to this file, we're going to do it in a separate PR with a few other FTR changes as well.